tools.py

```diff
@@ -0,0 +1,39 @@
+def interleave(*iterable):
+    size_min = 200
+    for i in iterable:
+        if len(i) < size_min:
+            size_min = len(i)
+
+    current_number = 0
+    while current_number < size_min:
+        for i in iterable:
+            yield i[current_number]
+        current_number = current_number + 1
+
+
+def main():
+    #example_1
+    arr = []
```

Comment:
https://www.w3schools.com/python/python_arrays.asp
arrays are not supported in python, lists are :)